static/js/publisher/release/constants.js

```diff
@@ -3,6 +3,7 @@ const CANDIDATE = "candidate";
 const BETA = "beta";
 const EDGE = "edge";
 const AVAILABLE = "AVAILABLE"; // fake 'risk' with all available revisions
+const BUILD = "BUILD"; // fake 'risk' for build sets
```

Comment:
Would be nice to get rid of those special case 'fake' risks. But this may require deeper refactor of release UI data structure and components...

Reply:
I agree, but it would probably mean adding another data type to each revision/ release